### PR TITLE
pandoc: install bash completions

### PR DESCRIPTION
**Pull request by leungbk**

### CODE REVIEW

**Comment by sternenseemann** [comment]
It's probably more maintenable to put these kind of things into nixpkgs in the future. Ref. #504

File: src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs [hunk 1]
```diff
@@ -396,6 +396,8 @@ pandocOverrides = set phaseOverrides postInstall
     postInstall = unlines [ "postInstall = ''"
                           , "  mkdir -p $out/share/man/man1"
                           , "  mv \"man/\"*.1 $out/share/man/man1/"
+                          , "  mkdir -p $out/share/bash-completion/completions"
+                          , "  mv data/bash_completion.tpl $out/share/bash-completion/completions/pandoc"
```
Comment (sternenseemann) [inline src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs:400]: Should probably use `pandoc --bash-completion` and thus needs to be tied to a check whether we are cross compiling, that's easier in nixpkgs.

**Comment by leungbk** [comment]
OK, I'll submit a PR to nixpkgs modifying `configuration-common.nix` instead.

**Comment by sternenseemann** [comment]
> OK, I'll submit a PR to nixpkgs modifying `configuration-common.nix` instead.

`configuration-nix.nix` for this sort of thing, please, but sounds good overall!

**Pull request closed without merging**